src/commands/rflib/logging/apex/instrument.ts

```diff
@@ -191,7 +189,87 @@ class ApexInstrumentationService {
   }
 
   public static processSystemDebugStatements(content: string, loggerName: string): string {
-    return content.replace(this.SYSTEM_DEBUG_REGEX, (match, debugMessage) => `${loggerName}.debug('${debugMessage}');`);
+    const debugStartRegex = /System\.debug\s*\(/g;
```

Comment:
All regex are defined as constants at the top. This pattern should be maintained.